style/stylesheets/betterttv-dark.css

```diff
@@ -1969,6 +1969,12 @@ color: white;
 
 .player-column .brick {
     background-color: #222;
+    border-left: 5px solid #6441a5 !important;
+	box-shadow: 0px 0 0 #6441a4 inset !important;
+}
+
+.what {
```

Comment:
`.what` is very nonspecific. What if that matches something else on the site and messes it up?